docs/start/getting-started/fragments/next/api.md

```diff
@@ -279,7 +279,9 @@ export default function Home({ posts = [] }) {
                 </fieldset>
 
                 <button>Create Post</button>
-                <button onClick={() => Auth.signOut()}>Sign out</button>
+                <button onClick={(e) => (e.preventDefault(), Auth.signOut())}>
```

Comment:
This is mostly just personal preference, but I think this might read a bit clearer. Your thoughts @ericclemmons?
 ```suggestion
                 <button
						         onClick={e => {
							           e.preventDefault();
							           Auth.signOut();
						         }}
                 >
```

Reply:
Great catch! I just tested the original behavior and don't know how I missed this 🤦 

```console
nonIterableSpread.js?3427:2 Uncaught (in promise) TypeError: Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.
    at _nonIterableSpread (nonIterableSpread.js?3427:2)
    at _toConsumableArray (toConsumableArray.js?2909:6)
    at _callee$ (VM5452 index.js:80)
    at tryCatch (runtime.js?96cf:63)
    at Generator.invoke [as _invoke] (runtime.js?96cf:293)
    at Generator.eval [as throw] (runtime.js?96cf:118)
    at asyncGeneratorStep (asyncToGenerator.js?1da1:3)
    at _throw (asyncToGenerator.js?1da1:29)
```

I would like to err on the side of _less_ JS and using the built-in HTML behavior:

```suggestion
                <button type="button" onClick={() => Auth.signOut()}>
                  Sign out
                </button>
```

This way, the first button is implicitly `type="submit"`, while the second is explicitly a `button type="button"`.